tests/StackExchange.Redis.Tests/Certificates/CertValidationTests.cs

```diff
@@ -0,0 +1,64 @@
+﻿using System;
+using System.IO;
+using System.Net.Security;
+using System.Security.Cryptography;
+using System.Security.Cryptography.X509Certificates;
+using System.Text;
+using System.Text.RegularExpressions;
+using Xunit;
+using Xunit.Abstractions;
+
+namespace StackExchange.Redis.Tests;
+
+public class CertValidationTests : TestBase
+{
+    public CertValidationTests(ITestOutputHelper output) : base(output) { }
+
+    [Fact]
+    public void CheckIssuerValidity()
+    {
+        // The endpoint cert is the same here
+        var endpointCert = LoadCert(Path.Combine("Certificates", "device01.foo.com.pem"));
```

Comment:
Instead of checking in certificates that will eventually expire; you could use System.Security.Cryptography.X509Certificates.CertificateRequest to just create them on the fly.  (net472+/net5+)